scripts/cat/cats.py

```diff
@@ -2028,7 +2028,11 @@ def get_injured(self, name, event_triggered=False, lethal=True, severity="defaul
                 )
                 != 0
             ):
-                clan_herbs = set(game.clan.herb_supply.entire_supply.keys())
+                clan_herbs = {
+                    herb
+                    for herb, herb_count in game.clan.herb_supply.entire_supply.items()
+                    if herb_count > 0
```

Comment:
```suggestion
                    if herb_count
```
Same here, since 0 is falsey and the herb supplies should never contain negatives, just checking `if herb_count` should be sufficient.  if that doesn't feel clear enough for future readers, you could renamed `herb_count` to `herbs_present`